programs/mango-v4/src/instructions/token_edit.rs

```diff
@@ -36,8 +36,9 @@ pub fn token_edit(
     deposit_weight_scale_start_quote_opt: Option<f64>,
     reset_stable_price: bool,
     reset_net_borrow_limit: bool,
-    reduce_only_opt: Option<bool>,
+    reduce_only_opt: Option<u8>,
     name_opt: Option<String>,
+    force_close_opt: Option<bool>,
```

Comment:
yeah, what do you mean by "collect"? collect what?

Reply:
If we wanted to parse instructions from the recorded transactions on-chain that'd be difficult because there are multiple versions of the same instruction with different arguments lists.